lib/presentation/pages/create_task_page.dart

```diff
@@ -25,17 +25,26 @@ class CreateTaskPage extends StatelessWidget {
     );
   }
 
+  Future _selectTime(BuildContext context) async {
+    TimeOfDay _currentTime = new TimeOfDay.now();
+    TimeOfDay selectedTime = await showTimePicker(
+        context: context,
+        initialTime: _currentTime,
+    );
+    if(selectedTime != null) print(selectedTime.toString());
+  }
```

Comment:
What do you think of moving the responsibility of presenting the picker to the component? I think it makes more sense

Reply:
I agree with you. 
I did it, and in this case, the same component can render different types, i made a enum for this.